cosmos/converter.py

```diff
@@ -138,6 +139,27 @@ def __init__(
             raise CosmosValueError(
                 "RenderConfig.dbt_project_path is required for rendering an airflow DAG from a DBT Graph if no manifest is provided."
             )
+        emit_datasets = render_config.emit_datasets
+        dbt_root_path = project_config.dbt_project_path.parent
+        dbt_project_name = project_config.dbt_project_path.name
+        dbt_models_dir = project_config.models_relative_path
+        dbt_seeds_dir = project_config.seeds_relative_path
+        dbt_snapshots_dir = project_config.snapshots_relative_path
+        test_behavior = render_config.test_behavior
+        select = render_config.select
+        exclude = render_config.exclude
+        dbt_deps = render_config.dbt_deps
+        execution_mode = execution_config.execution_mode
+        load_mode = render_config.load_method
+        manifest_path = project_config.parsed_manifest_path
+        dbt_executable_path = execution_config.dbt_executable_path
+        node_converters = render_config.node_converters
```

Comment:
@LennartKloppenburg I think (most of) these were added by mistake as part of rebasing - and they are causing the errors during the unit tests.

These lines have been deleted during some recent refactoring, including 2f8d0e2f787a5792feacba609c0190877062375c and b64eb9a1f665a44327c6a939c27334b7c9da084b.